src/nimble.nim

```diff
@@ -8,17 +8,17 @@ import std/options as std_opt
 
 import strutils except toLower
 from unicode import toLower
-
+import sat/sat
```

Comment:
Now or later, you need to ensure `koch` bundles `nimble` correctly, which needs to clone `sat` instead.